vmm/Cargo.toml

```diff
@@ -28,7 +28,7 @@ event_monitor = { path = "../event_monitor" }
 hypervisor = { path = "../hypervisor" }
 lazy_static = "1.4.0"
 libc = "0.2.95"
-linux-loader = { version = "0.3.0", features = ["elf", "bzimage", "pe"] }
+linux-loader = { git = "https://github.com/michael2012z/linux-loader", branch = "aarch64-uefi", features = ["elf", "bzimage", "pe"] }
```

Comment:
This is temporary for introducing the new interface in `vm.rs`.

Reply:
Please mark this PR as a draft until your PR on the linux loader gets merged.